src/AMP_for_E-Commerce%3A_Getting_Started.html

```diff
@@ -0,0 +1,185 @@
+  left: 31px;
+  right: 30px;
+}
+  </style>
+</head>
+<body>
+  <!-- 
+    ## Product pages
+
+    It's possible with AMP to create beautiful and interactive product pages. [This sample](/samples_templates/product_page/preview/) demonstrates how to build a complex product page with dynamic product configuration and add-to-cart flow. With the introduction of [amp-bind](/components/amp-bind/) it's now possible to create truly interactive AMP pages: : it can be used to coordinate page state based on user interaction to show and hide arbitrary divs. 
+
+
+    <div class="ampstart-mobile-frame xs-hide relative">
+      <amp-img src="https://ampstart.com/img/www/mobile.svg" width="458" height="806" alt="Mobile outline" layout="responsive" class="absolute top-0 left-0 right-0 bottom-0"></amp-img>
+
+      <amp-iframe width="324" height="494" layout="responsive" sandbox="allow-scripts allow-popups" allowfullscreen="" frameborder="0" src="/samples_templates/product_page/preview/embed/" class="absolute xs-hide">
+      <div placeholder=""></div>
+      </amp-iframe>
+    </div>
+
+
+    These samples help you getting started with building a product landing page with AMP: 
+
+    * **Product gallery:** with [amp-carousel](/components/amp-carousel/) and [amp-bind](/components/amp-bind) it is easy to create sophisticated image galleries. [Here](/advanced/image_galleries_with_amp-carousel/) are some samples how to implement image gallieries with captions and thumbnails in AMPHTML.
+    * **Product configuration:** use [amp-selector](/components/amp-selector) and [amp-bind](/components/ampbind) for advanced product configuration. For an advanced version see the [sample product page](/samples_templates/product_page/preview/).
+    * **Add to Cart:** the [product sample](/samples_templates/product_page/preview/) demonstrates a fully functional add-to-cart flow which works across different origins.
+    * **Tabs:** can be easily implemented by combining [amp-selector](/components/amp-selector/) with a flex layout. [Here](/advanced/tab_panels_with_amp-selector/) is a sample.
+    * **Star ratings:** [here](/advanced/star_rating/) is a sample demonstrating how to implement a star rating system in AMP. 
+    * **Call tracking:** use [amp-call-tracking](/components/amp-call-tracking) to track calls initiated from your AMPs.
+    * **Comments / reviews:** comment or review submission can be implemented using [amp-form](/components/amp-form). Combine with [amp-access](/components/amp-access/) if a user login is required. [Here](/samples_templates/comment_section/preview/) is a sample combining both.
+
+
+    ## Category Pages
+
+    Product category pages are very well suited for AMP. The content is typically static and geared toward offering the best showcase of available products. [Here](/samples_templates/product_browse_page/preview/) is a working sample of a product pages demonstrating common features of a category pages such as product listings or search.
```

Comment:
"The content is typically static" ← I think this should be removed. I don't have an ideal replacement though. Maybe something along the lines of, "As popular landing pages for users, product category pages are well suited for AMP. They are often a mix of editorial content and the hero-style presentation of products."